bzt/modules/proxy2jmx.py

```diff
@@ -16,71 +16,73 @@
 limitations under the License.
 """
 
-import json
 import sys
 import time
 import os
 import shutil
 
-import requests
+from bzt.bza import BZAObject
 
 from os.path import join, isfile
 
-from bzt import TaurusConfigError, TaurusNetworkError, TaurusInternalException
+from bzt import TaurusConfigError, TaurusInternalException
 from bzt.engine import Service
 from bzt.utils import is_windows, get_full_path
 from bzt.modules.selenium import AbstractSeleniumExecutor
 
 
-class Proxy2JMX(Service):
+class BZAProxy(BZAObject):
     def __init__(self):
-        super(Proxy2JMX, self).__init__()
-        self.proxy = None
-        self.headers = {}
-        self.api_delay = 5
-        self.address = 'https://a.blazemeter.com/api/latest/mitmproxies'
-        self.label = 'generated'
+        super(BZAProxy, self).__init__()
+        self.address = 'https://a.blazemeter.com/api/latest/proxy'
```

Comment:
You should rely on base address, same way other classed do. This provides common approach to env switching and unit testing.